ts/test/session/sending/PendingMessageCache_test.ts

```diff
@@ -68,25 +67,66 @@ describe('PendingMessageCache', () => {
     expect(addedMessage.timestamp).to.deep.equal(rawMessage.timestamp);
   });
 
+  it('can add multiple messages belonging to the same user', async () => {
+    const device = TestUtils.generateFakePubKey();
+
+    await pendingMessageCacheStub.add(device, TestUtils.generateChatMessage());
+    // We have to timeout here otherwise it's processed too fast and messages start having the same timestamp
```

Comment:
Good idea

Reply:
Took me a while to figure it out, i was wondering why tests were failing when they used to pass before